test/tests/functional/pbs_qmgr.py

```diff
@@ -159,3 +159,25 @@ def test_multi_attributes(self):
             self.logger.info("looking for error, %s" % msg)
             self.assertTrue(msg in e.msg[0])
         self.server.expect(NODE, 'queue', op=UNSET, id=self.mom.shortname)
+
+    def test_string_quoting(self):
+        """
+        Test to verify that if a string attribute has spaces, the value
```

Comment:
@vstumpf , should we also a test to address the case where string attribute has quotes ?